lib/utils/execution-state-manager.js

```diff
@@ -13,8 +13,8 @@ class ExecutionStateManager {
     // A map of browerId to test modules executed for the current test execution.
     this._browserToModuleMap = new Map();
 
-    // An array keeping the module execution details
-    this._moduleMetadata = [];
+    // An map keeping the module execution details
```

Comment:
'A map', not 'An map'. Also, what is the content of the array - individual test metadata for 1 module? If so, maybe this could be _testMetadata?

Reply:
Content will be a test module e.g.
[{ moduleName, totalTest, skippedTest, FailedTest, duration}...]